core/shared/utils/uncommon/bh_getopt.c

```diff
@@ -0,0 +1,55 @@
+#ifndef __GNUC__
```

Comment:
Missing license header

Reply:
Added, btw, the license header may need to be added for the bh_read_file.c also under the same directory.